### DISCUSSION TITLE
adguardhome

### DISCUSSION THREAD

**Discussion by neilo1** [wulfy23/rpi4#27] [Ideas]
Hello,

Is it possible to add adguard home in pi4 image? Is it better compared to the packages we are using in the built for adblocking and dns over https? 
https://forum.openwrt.org/t/how-to-updated-2021-installing-adguardhome-on-openwrt-manual-and-opkg-method/113904/400

https://openwrt.org/docs/guide-user/services/dns/adguard-home

Thank you,

**Comment by wulfy23** [comment]
Hello,

Did not see you'd popped the discussions cherry. Tell you what, lemme simplify my earlier response in the related issue. If two other people would regularly use this AND all share similar DNS topology ( with provided functional configs ) I will make it happen.

Due to the size i'll need to get hosting (tho' it would not be the only reason or beneficiary) and provide a separate channel ('flavour') / image.

Cheers

**Comment by neilo1** [comment]
Yeah, I thought you have not seen this discussion post, thats why I opened an issue. I was looking into adguard home when I was facing issue with DNS. I installed the adguardhome but it was not auto starting after the reboot, and you fixed dns issue for me. So, I stopped after that.

**Comment by neilo1** [comment]
Thank you for the new update.
